store/CSDK/main.cpp

```diff
@@ -168,9 +220,51 @@ bool readFromS3(JNIEnv *env, char *argv[]) {
         printf("\n");
     }
 
+    printf("\nread Schema from Index File from S3:\n");
+    CarbonSchemaReader carbonSchemaReader(env);
+    jobject schema = carbonSchemaReader.readSchemaInIndexFile(
+            "s3a://sdk/WriterOutput/carbondata/510199997055746_batchno0-0-null-510199277323454.carbonindex");
```

Comment:
do we have these binaries part-0-510199997055746_batchno0-0-null-510199277323454.carbondata? I think we should not keep any binaries in repo.

Reply:
no binaries files in repo, only in my local